W/wannier90/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "wannier90"
+version = v"3.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/wannier-developers/wannier90/archive/v3.1.0.tar.gz", "40651a9832eb93dec20a8360dd535262c261c34e13c41b6755fa6915c936b254")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wannier90-*/
+
+shared="-shared"
+if [[ "${target}" == *-apple-* ]]; then
+    shared="-dynamiclib"
+fi
+cat > make.inc << EOF
+    F90    = $FC
+    FCOPTS = $FFLAGS -fPIC -O2
```

Comment:
You can use `-O3` if you want, what you can't use is `-march`
```suggestion
    FCOPTS = $FFLAGS -fPIC -O3
```

Reply:
I tried and it caused issues on my local built by architecture-dependent optimisations. Will that not occur on the build farm as well?